exercises/practice/meetup/create_fixture.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS meetup;
+CREATE TABLE "meetup" (
+    "year" INTEGER,
+    "month" INTEGER,
+    "week" TEXT,
+    "dayofweek" TEXT,
+    "result" TEXT
+);
+
+-- Note: the CSV file contain literal tab, newline, carriage returns.
```

Comment:
Is this comment true?

Reply:
As I used the nifty bash script, I just took what was there.
Maybe we should use
```suggestion
-- Note: the CSV file may contain literal tab, newline, carriage returns.
```